phalcon/Config/ConfigFactory.zep

```diff
@@ -80,7 +74,7 @@ class Factory extends BaseFactory
             );
         }
 
-        let className = namespaceClass . "\\" . camelize(adapter);
+        let className = "Phalcon\\Config\\Adapter\\" . camelize(adapter);
```

Comment:
What about user defined classes?

Reply:
This was never designed for custom classes for config. The old `load` method would load only a prefix namespace and then one of the adapters